explainer - Emisie dychanie

LenkaSedlackova opened the pull request.

rastislavsk committed:
Explainer: emisie-dychanie, preklad

LenkaSedlackova committed:
jazykový check

mukrop committed:
Zmeny nutné pre prvý explainer

* Povolené explainery v hlavnom configu.
* Dočasne pridaný redirect a anchor /o-nas#members na titulnú stránku kvôli automatickému prekliku z hlavičky explainerov.
* Pridaný preklad sprievodnej vety k explainerom na stránke témy.
* Zmazaný placeholder explainer.
* Pridaný cover obrázok pre explainer.
* Aktualizovaný web-core.

mukrop reviewed (approved):
Dobra praca!

Kedze je to prvy slovensky explainer, bolo tam viacero veci na dokoncenie, zapnutie, upravu, ale mame ho :-).

mukrop committed:
Pridaná aktualita

Publikovaná infografika "História výskumu skleníkového efektu" a explainer o dýchaní.
